crates/runtime/src/vmcalls/transfer.rs

```diff
@@ -0,0 +1,46 @@
+use crate::FuncEnv;
+use svm_state::AccountStorage;
+use svm_types::{Address, BytesPrimitive};
+
+fn load_addr(env: &FuncEnv, mem_ptr: u32) -> Address {
+    let borrow = env.borrow();
+    let wasm_memory = &borrow.memory().view::<u8>();
+
+    let byte_cells = &wasm_memory[mem_ptr as usize..mem_ptr as usize + Address::N];
+    let mut bytes = [0u8; Address::N];
+    for (i, cell) in byte_cells.iter().enumerate() {
+        bytes[i] = cell.get();
+    }
+    Address::new(bytes)
+}
+
+/// Sends coins from the current executing account to a destination account.
+///
+/// # Panics
+///
+/// Panics when the destination account does not exist.
+pub fn svm_transfer(env: &FuncEnv, src_addr_ptr: i32, dst_addr_ptr: i32, amount: i64) {
+    let borrow = env.borrow();
+    let gs = borrow.storage().gs.clone();
+
+    let src_addr = load_addr(env, src_addr_ptr as u32);
```

Comment:
The `load_addr` could receive `i32` or `WasmPtr` instead of `u32`

Reply:
`WasmPtr` is nicer, didn't think about it. Fixing now.